presets/4.3/tune/Tehllama_533_TinyTrainer_3in_3S_TUNE_Options.txt

```diff
@@ -0,0 +1,151 @@
+#$ TITLE: FlyFive33 3in TinyTrainer 3S Tune
+#$ FIRMWARE_VERSION: 4.3
+#$ CATEGORY: TUNE
+#$ STATUS: EXPERIMENTAL
+#$ KEYWORDS: TinyTrainer, 533, Spec, Race, Llama
+#$ AUTHOR: Daniel Appel / Tehllama && Armando Gallegos / Mondo && Evan Turner / HeadsUp
+#$ DESCRIPTION: This is an adaptation of the 533 TinyTrainer tune, and expects bidirectional DShot ESCs at 48kHz PWM with MedHigh timing. 
+#$ DESCRIPTION: These fly best on 3S batteries in the 450-550mAh capacity range with 533 1404 motors.
+#$ DISCUSSION: https://github.com/betaflight/firmware-presets/pull/233
+
+#$ INCLUDE: presets/4.3/tune/defaults.txt
+# name: FIVE33
+
+# master
+
+set dshot_idle_value = 700
+# Fallback if Dynamic Idle not working or not selected
+set dshot_bidir = ON
```

Comment:
this should go inside filter potions as it is an option from filters.
Also setting dshot_bidir = ON won't work properly without making sure the ESC protocol is set to DSHOT300/600.
YOu can set ESC protocol to dshot300 or 600 in your filters options, but then you should include #$ WARNING_INCLUDE about dshot like in some other presets.

Reply:
Relocated to RPM Features.
Will regression test with Default/Noisy/Clean no-RPMF Presets